op-program/client/l2/engineapi/l2_engine_api.go

```diff
@@ -565,3 +600,15 @@ func (ea *L2EngineAPI) invalid(err error, latestValid *types.Header) *eth.Payloa
 	errorMsg := err.Error()
 	return &eth.PayloadStatusV1{Status: eth.ExecutionInvalid, LatestValidHash: &currentHash, ValidationError: &errorMsg}
 }
+
+// convertRequests converts a hex requests slice to plain [][]byte.
+func convertRequests(hex []hexutil.Bytes) [][]byte {
+	if hex == nil {
+		return nil
+	}
+	req := make([][]byte, len(hex))
+	for i := range hex {
+		req[i] = hex[i]
+	}
+	return req
+}
```

Comment:
I was wondering if its possible to do this in a 1 liner but doesn't seem possible without `unsafe`

```go
// Warning: not recommended for production.
func convertRequestsUnsafe(hex []hexutil.Bytes) [][]byte {
	return *(*[][]byte)(unsafe.Pointer(&hex))
}
```